src/test/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentServiceSpec.scala

```diff
@@ -1,213 +1,145 @@
-    val fileMetadataRepositoryMock = mock[FileMetadataRepository]
-    val fileRepositoryMock = mock[FileRepository]
-    val ffidMetadataRepositoryMock = mock[FFIDMetadataRepository]
-    val fixedUuidSource = new FixedUUIDSource()
-
-    val service: ConsignmentService = new ConsignmentService(consignmentRepoMock,
-      fileMetadataRepositoryMock,
-      fileRepositoryMock,
-      ffidMetadataRepositoryMock,
-      FixedTimeSource,
-      fixedUuidSource)
-
-    val consignmentId = UUID.fromString("3c8da55a-bca0-4cd8-8efb-fb2d316e88ee")
     val filesProcessed = 78
-
     when(fileRepositoryMock.countProcessedAvMetadataInConsignment(consignmentId)).thenReturn(Future.successful(filesProcessed))
     when(fileMetadataRepositoryMock.countProcessedChecksumInConsignment(consignmentId)).thenReturn(Future.successful(filesProcessed))
     when(ffidMetadataRepositoryMock.countProcessedFfidMetadata(consignmentId)).thenReturn(Future.successful(filesProcessed))
 
-    val progress: FileChecks = service.getConsignmentFileProgress(consignmentId).futureValue
-
+    val progress: FileChecks = consignmentService.getConsignmentFileProgress(consignmentId).futureValue
     progress.antivirusProgress.filesProcessed shouldBe filesProcessed
     progress.checksumProgress.filesProcessed shouldBe filesProcessed
     progress.ffidProgress.filesProcessed shouldBe filesProcessed
   }
 
   "getConsignmentParentFolder" should "return the parent folder name for a given consignment" in {
-    val consignmentRepoMock = mock[ConsignmentRepository]
-    val fileMetadataRepositoryMock = mock[FileMetadataRepository]
-    val fileRepositoryMock = mock[FileRepository]
-    val ffidMetadataRepositoryMock = mock[FFIDMetadataRepository]
-    val fixedUuidSource = new FixedUUIDSource()
-
-    val service: ConsignmentService = new ConsignmentService(consignmentRepoMock,
-      fileMetadataRepositoryMock,
-      fileRepositoryMock,
-      ffidMetadataRepositoryMock,
-      FixedTimeSource,
-      fixedUuidSource)
-
-    val consignmentId = UUID.fromString("d8383f9f-c277-49dc-b082-f6e266a39618")
     val parentFolder: Option[String] = Option("CONSIGNMENT SERVICE PARENT FOLDER TEST")
-
     when(consignmentRepoMock.getParentFolder(consignmentId)).thenReturn(Future.successful(parentFolder))
 
-    val parentFolderResult: Option[String] = service.getConsignmentParentFolder(consignmentId).futureValue
-
+    val parentFolderResult: Option[String] = consignmentService.getConsignmentParentFolder(consignmentId).futureValue
     parentFolderResult shouldBe parentFolder
   }
+
+  "getSeriesOfConsignment" should "return the series for a given consignment" in {
+    val mockSeries = Seq(SeriesRow(seriesId, bodyId, seriesName, seriesCode, seriesDescription))
+    when(consignmentRepoMock.getSeriesOfConsignment(consignmentId)).thenReturn(Future.successful(mockSeries))
+
+    val expectedSeries: SeriesFields.Series = consignmentService.getSeriesOfConsignment(consignmentId).futureValue.get
```

Comment:
Variable naming: this could just be `series`. It's not the expected series, it's the actual one that the service returns.

Reply:
Ohhh, I see; I thought "expected" meant what you expect to receive from the service.